src/Components/Article/ArticleCard.jsx

```diff
@@ -1,10 +1,11 @@
 import { Link } from "react-router-dom";
 import VoteComponent from "../Vote";
+import CommentList from "../Comment/CommentList";
 
 const ArticleCard = ({ article }) => {
   return (
     <Link to={`/articles/${article.article_id}`}>
-      <section className="max-w-lg mx-auto rounded overflow-hidden shadow-lg bg-white p-4 m-4 transition-transform transform hover:scale-105 max-w-screen-xl mx-auto px-4 py-8">
+      <section className="max-w-screen-xl mx-auto rounded overflow-hidden shadow-lg bg-white p-4 m-4 transition-transform transform hover:scale-105 px-4 py-8">
```

Comment:
I'd reconsider the hover transformation, your article cards are very large (not a bad thing) but because they take up most of the screen this hover is unavoidable and a little hard to navigate around from a UX perspective. Same goes for the comment cards, what does the hover transformation imply? What is the goal?